Tasks/Maven/maventask.ts

```diff
@@ -6,15 +6,15 @@ import fs = require('fs');
 import tl = require('vsts-task-lib/task');
 import trm = require('vsts-task-lib/toolrunner');
 
-import pmd = require('./pmdForMaven');
-import ar = require('./analysisResult');
+import pmd = require('./pmdformaven');
+import ar = require('./analysisresult');
```

Comment:
Done, though no further context could be found for "xplat issues" from the test that failed.